lib/components/Button/Button.tsx

```diff
@@ -1,10 +1,8 @@
-import classNames from 'classnames';
-/**
- * base = default button to be used
- * primary = should be only appear once in a page
- * text = for less-pronounced actions
- */
-export type ButtonVariant = 'base' | 'primary' | 'outlined' | 'text';
+export type ButtonVariant =
+  | 'base' // default button to be used
+  | 'primary' // should be only appear once in a page
+  | 'outlined'
+  | 'text'; // for less-pronounced actions
```

Comment:
This will not show up in the JSDoc, but on the other hand the previous one was not so readable so I'm fine with the change. Maybe later on communicating these in documentation on Storybook.
And this whole Button component is just a lucky guess what the component can later be.